src/controllers/update/beneficial-owner-bo-mo-review.controller.ts

```diff
@@ -0,0 +1,34 @@
+import { NextFunction, Request, Response } from "express";
+
+import { logger } from "../../utils/logger";
+import * as config from "../../config";
+import { ApplicationData } from "model";
+import { getApplicationData } from "../../utils/application.data";
+
+export const get = (req: Request, res: Response, next: NextFunction) => {
+  try {
+    logger.debugRequest(req, `${req.method} ${req.route.path}`);
+
+    const appData: ApplicationData = getApplicationData(req.session);
+
+    return res.render(config.BENEFICIAL_OWNER_BO_MO_REVIEW_PAGE, {
+      backLinkUrl: config.UPDATE_REGISTRABLE_BENEFICIAL_OWNER_PAGE,
```

Comment:
URL not page please. backLinkUrl is used for deciding if it's update journey so does need to be a somethingLinkUrl